package.json

```diff
@@ -142,6 +142,7 @@
     "scaffold": "./node_modules/.bin/gulp ensureConfig && ./node_modules/.bin/gulp ensureDevProxy",
     "start": "npm run check-env && NODE_ENV=development webpack-dev-server --config ./webpack/webpack.dev.babel.js --hot --progress --colors --content-base ./build --host 0.0.0.0 --port $npm_package_config_port",
     "test": "node jest/gen-config.js && NODE_PATH=node_modules jest --config=jest/config.json --no-cache",
+    "test:watch": "npm test -- --watch",
```

Comment:
Hey @weblancaster - thanks for this. Think it makes it more explicit for those who don't know you can do `npm run test -- --watch`